src/test/java/fitr/common/CommandsTest.java

```diff
@@ -0,0 +1,20 @@
+package fitr.common;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class CommandsTest {
+    @Test
+    public void commandsTest_correctResult() {
+        assertEquals("bye",Commands.COMMAND_BYE);
+        assertEquals("food", Commands.COMMAND_FOOD);
+        assertEquals("exercise", Commands.COMMAND_EXERCISE);
+        assertEquals("goal", Commands.COMMAND_GOAL);
+        assertEquals("help", Commands.COMMAND_HELP);
+        assertEquals("recommend", Commands.COMMAND_RECOMMEND);
+        assertEquals("view", Commands.COMMAND_VIEW);
+        assertEquals("delete", Commands.COMMAND_DELETE);
+        assertEquals("edit", Commands.COMMAND_EDIT);
```

Comment:
I think this JUnit test is quite trivial?

Reply:
It is trivial but i think the user may changes the messages.

Reply:
The commands are all static final already